test/docker-tests.robot

```diff
@@ -101,7 +101,7 @@ Single node compose example
   Verify response for authenticated request with  8001  *No license key has been entered*
   Verify response for authenticated request with  8002  *Monitoring Dashboard*
   Compose logs should contain  ../docker-compose/marklogic-centos.yaml  *Setting timezone to Europe/Prague*
-  [Teardown]  Delete compose from  ../docker-compose/marklogic-centos.yaml
+  [Teardown]  Run Keywords  Delete compose from  ../docker-compose/marklogic-centos.yaml  AND  Remove Files  mldb_admin_username.txt  mldb_admin_password.txt  marklogic-centos.yaml
```

Comment:
I think this complicates the test cases a bit. We don't need to worry about clean up of specific files on test case level. It should just be a function of "Delete compose" or some other clean up keyword.